src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void testIsAnyNull() {
+        assertTrue(Utils.isAnyNull("asdf", 1234, null));
+    }
+    
+    @Test
+    public void testNotIsAnyNull() {
+        assertFalse(Utils.isAnyNull("asdf", 1234));
+    }
+    
+    @Test 
+    public void testElementsAreUnique() {
+        ArrayList<Integer> testList = new ArrayList<Integer>();
+        for (int i = 0; i < 5; i++) {
+            testList.add(i);
+        }
```

Comment:
You can use `List<Integer> list = Arrays.asList(1, 1, 1, 2)` to create and initialize a list in one line.